container-setup/install/install-kube_ps1.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+if [ "$I_AM_IN_CONTAINER" != "I-am-in-container" ]; then
+  echo "must be run in container";
+  exit 1;
+fi
+
+echo "in container";
+
+mkdir /usr/local/kube_ps1;
+pushd /usr/local/kube_ps1;
+wget -q https://raw.githubusercontent.com/drewandersonnz/kube-ps1/master/kube-ps1.sh;
```

Comment:
This should pull either directly from the upstream kube-ps1, or use a copy forked in this repo, just in case (heaven forbid) Drew decides to leave us someday.

Reply:
I agree with having it not tied to me, it is a fork in my repo so we can check any changes ... I don't want to reply on someone external to our team for a security review of inbound code.

Last thing we want is an attacker gaining access to and submitting dangerous code to a smaller repo that we depend on.

I'm less concerned about other tools in the `github.com/openshift` organisation, as we are in control of that.